deepspeed/runtime/engine.py

```diff
@@ -632,7 +632,10 @@ def _configure_fp16_optimizer(self, optimizer):
         dynamic_loss_args = self.dynamic_loss_scale_args()
         clip_grad = self.gradient_clipping()
         if isinstance(optimizer,
-                      FusedAdam) or self.optimizer_name() == ONEBIT_ADAM_OPTIMIZER:
+                      FusedAdam) or self.optimizer_name() in [
+                          'FusedAdam',
```

Comment:
Maybe we can do this in a similar way as what we're doing with zero compatible optimizers. As in, attempt to import apex and append apex's FusedAdam to a list. I see that `isinstance` accepts a tuple as a 2nd arg which should make it a bit easier to check across multiple classes. 

The annoying part is we still have to check 1bit-adam via string since if we import the class it currently requires MPI, hopefully we can clean that up in the upcoming changes using NCCL so there are no dependency issues simply from the import (cc: @awan-10, @conglongli).

https://github.com/microsoft/DeepSpeed/blob/master/deepspeed/runtime/zero/utils.py#L26-L38

Reply:
Sounds good, I pushed an update.